sql/server.go

```diff
@@ -175,6 +193,13 @@ func (s *Server) exec(req driver.Request) (resp driver.Response, err error) {
 		if err = gogoproto.Unmarshal(req.Session, &planMaker.session); err != nil {
 			return
 		}
+		if planMaker.session.Txn != nil {
+			planMaker.txn = s.db.ContinueTxn(*planMaker.session.Txn)
```

Comment:
I can see why you would need to save the transaction state in the defer, but why do you need to continue the transaction here?